src/quests.cpp

```diff
@@ -232,3 +269,45 @@ bool Quests::isQuestStorage(const uint32_t key, const int32_t value, const int32
 	}
 	return false;
 }
+
+void Quests::createQuest(Quest* quest)
```

Comment:
Make it an `unique_ptr`, so it symbolizes that a pointer passed to this function must not be used by the caller (it changes ownership).

Reply:
I've been trying to do what you suggested, but to no avail.
Any attempt to wrap `Quest*` in a unique_ptr resulted in a `C2440` error.

I made some changes, though I don't know if it's still really necessary to wrap Quest* in unique_ptr?